packages/client/services/AuthenticationService.ts

```diff
@@ -0,0 +1,13 @@
+import { SignUpResponseBody } from '@musicroom/types';
+import redaxios from 'redaxios';
+import urlcat from 'urlcat';
+import { SERVER_ENDPOINT } from '../constants/Endpoints';
+
+export async function sendSignUp(): Promise<SignUpResponseBody> {
+    const url = urlcat(SERVER_ENDPOINT, '/authentication/sign-up');
+
+    const rawResponse = await redaxios.post(url);
+    const parsedResponse = SignUpResponseBody.parse(rawResponse.data);
+
+    return parsedResponse;
+}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏It seems that implementing authentication is really near us 👀.